Xamarin.Forms.Core.UITests.Shared/Tests/ToolbarItemTests.cs

```diff
@@ -11,11 +12,11 @@ namespace Xamarin.Forms.Core.UITests
 	[Category(UITestCategories.UwpIgnore)]
```

Comment:
I think we might disable these tests for now, remove the asserts, take screenshots and mark as manual review